packages/blade/src/components/Amount/_decisions/decisions.md

```diff
@@ -0,0 +1,73 @@
+# Amount <!-- omit in toc -->
+
+Amount component is a UI element that displays and formats various currency values in a specified format.
+
+<img  src="./amount-thumbnail.png"  width="380" />
+
+## Design
+
+- [Figma - Amount](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=28012%3A580578&t=3peAz8A2n2Gw4WMl-1)
+
+## API
+
+| Prop                      | Type                                                       | Default     | Description                                                                                                                                                                                       | Required |
+| ------------------------- | ---------------------------------------------------------- | ----------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------- |
+| value                     | `number`                                                   | `undefined` | The value to be rendered within the component                                                                                                                                                     | ✅       |
+| variant                   | `positive`, `negative`, `notice`, `information`, `blue`    | `undefined` | The variant of the amount to be rendered                                                                                                                                                          |          |
+| fontWeight                | `bold`, `regular`                                          | `bold`      | The weight of the amount to be rendered                                                                                                                                                           |          |
+| showSuffix                | `none`, `decimals` , `humanise`                            | `decimals`  | The type of affix to be displayed with the currency value, (none, decimals, and humanise) that respectively indicate no affix, the number of decimal places, and the scale of the currency value. |          |
```

Comment:
Since this isn't a boolean anymore we can avoid the `show` prefix and just make the prop `suffix` perhaps:

```ts
suffix: 'none' | 'decimals' | 'humanise'
```

One thing to confirm whether to opt for `humanise` (british) or `humanize` (american) 😸

Reply:
+1

I'd vote for `humanize`